content/en/agent/kubernetes/_index.md

```diff
@@ -174,11 +174,155 @@ To install the Datadog Agent on your Kubernetes cluster:
 {{% /tab %}}
 {{% tab "Operator" %}}
 
-[The Datadog Operator][1] is in public beta. The Datadog Operator is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options. To get started, check out the [Getting Started page][2] in the [Datadog Operator repo][1] or install the operator from the [OperatorHub.io Datadog Operator page][3].
+[The Datadog Operator][1] is in public beta. The Datadog Operator is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
 
-[1]: https://github.com/DataDog/datadog-operator/blob/master/docs/getting_started.md
-[2]: https://github.com/DataDog/datadog-operator
-[3]: https://operatorhub.io/operator/datadog-operator
+## Prerequisites
+
+Using the Datadog Operator requires the following prerequisites:
+
+- **Kubernetes Cluster version >= v1.14.X**: Tests were done on versions >= `1.14.0`. Still, it should work on versions `>= v1.11.0`. For earlier versions, because of limited CRD support, the operator may not work as expected.
+- [`Helm`][2] for deploying the `datadog-operator`.
+- [`Kubectl` CLI][3] for installing the `datadog-agent`.
+
+## Deploy the Datadog Operator
```

Comment:
We have a very important goal in mind - to make the Operator the recommended way to deploy the Datadog Agent (over Helm and DaemonSet). In order to get there, one of the key requirements is to make the Operator easier and quickest to deploy. There are too many steps below that are true, but provide too much information than a quick "getting started" guide. We actually designed and implemented the quick start path that is documented here: https://github.com/DataDog/datadog-operator/blob/master/docs/getting_started.md#deploy-an-agent-with-the-operator

My suggestion is to use the steps there to deploy the Operator+Agent as in few steps as possible, and move the more advanced steps that offer additional customization to a different location in the page or another page

Reply:
We can move the verbose steps below to the first part of the Operator Configurations page, and add a link after the quick start steps in this page with a link called "additional/advanced configuration", wdyt?